packages/babel-standalone/src/index.js

```diff
@@ -0,0 +1,320 @@
+  "transform-es2015-template-literals": require("babel-plugin-transform-es2015-template-literals"),
+  "transform-es2015-typeof-symbol": require("babel-plugin-transform-es2015-typeof-symbol"),
+  "transform-es2015-unicode-regex": require("babel-plugin-transform-es2015-unicode-regex"),
+  "transform-es3-member-expression-literals": require("babel-plugin-transform-es3-member-expression-literals"),
+  "transform-es3-property-literals": require("babel-plugin-transform-es3-property-literals"),
+  "transform-es5-property-mutators": require("babel-plugin-transform-es5-property-mutators"),
+  "transform-eval": require("babel-plugin-transform-eval"),
+  "transform-exponentiation-operator": require("babel-plugin-transform-exponentiation-operator"),
+  "transform-export-extensions": require("babel-plugin-transform-export-extensions"),
+  "transform-flow-comments": require("babel-plugin-transform-flow-comments"),
+  "transform-flow-strip-types": require("babel-plugin-transform-flow-strip-types"),
+  "transform-function-bind": require("babel-plugin-transform-function-bind"),
+  "transform-jscript": require("babel-plugin-transform-jscript"),
+  "transform-object-assign": require("babel-plugin-transform-object-assign"),
+  "transform-object-rest-spread": require("babel-plugin-transform-object-rest-spread"),
+  "transform-object-set-prototype-of-to-assign": require("babel-plugin-transform-object-set-prototype-of-to-assign"),
+  "transform-proto-to-assign": require("babel-plugin-transform-proto-to-assign"),
+  "transform-react-constant-elements": require("babel-plugin-transform-react-constant-elements"),
+  "transform-react-display-name": require("babel-plugin-transform-react-display-name"),
+  "transform-react-inline-elements": require("babel-plugin-transform-react-inline-elements"),
+  "transform-react-jsx": require("babel-plugin-transform-react-jsx"),
+  "transform-react-jsx-compat": require("babel-plugin-transform-react-jsx-compat"),
+  "transform-react-jsx-self": require("babel-plugin-transform-react-jsx-self"),
+  "transform-react-jsx-source": require("babel-plugin-transform-react-jsx-source"),
+  "transform-regenerator": require("babel-plugin-transform-regenerator"),
+  "transform-runtime": require("babel-plugin-transform-runtime"),
+  "transform-strict-mode": require("babel-plugin-transform-strict-mode"),
+  "undeclared-variables-check": require("babel-plugin-undeclared-variables-check"),
+});
+
+// All the presets we should bundle
+registerPresets({
+  es2015: require("babel-preset-es2015"),
+  es2016: require("babel-preset-es2016"),
+  es2017: require("babel-preset-es2017"),
+  react: require("babel-preset-react"),
+  "stage-0": require("babel-preset-stage-0"),
+  "stage-1": require("babel-preset-stage-1"),
+  "stage-2": require("babel-preset-stage-2"),
+  "stage-3": require("babel-preset-stage-3"),
+
+  // ES2015 preset with es2015-modules-commonjs removed
+  // Plugin list copied from babel-preset-es2015/index.js
+  "es2015-no-commonjs": {
```

Comment:
@hzoo - I kept everything as-is just to maintain backwards compatibility with older versions of babel-standalone. Perhaps we could make `es2015-no-commonjs` into a "fake" preset that passes in the correct options. Let's do that separately though 😃